test/controllers/users/registrations_controller_test.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+module Users
+  class RegistrationsControllerTest < ActionDispatch::IntegrationTest
+    test "should register new user" do
+      user = {
+        name: "name",
+        surname: "surname",
+        email: "test@example.com",
+        password: "password",
+        password_confirmation: "password"
+      }
+
+      post "/user", params: {
+        user: user
+      }
+
+      user_list = User.find_by!(name: user[:name], surname: user[:surname], email: user[:email])
+      assert user_list
```

Comment:
I think you should use `assert_not_nil` here, so it is more clear what you do
https://guides.rubyonrails.org/testing.html#available-assertions